### ISSUE TITLE
Missing attachment in email

### ISSUE THREAD

**Issue by mfreminville** [nylas/N1#2336]
It seems that some attachment are missing in some emails. I've received 2 emails last week where at least one attachment is missing (one xlsx file and one pptx file). 
I'm on Mac OSX, using exchange. When I open the mail in an other mail client, I'm able to see the attachment but it's missing in N1.

What information could I provide to help you figure out what's the issue ?

Thanks

**Comment by pscheid92** [comment]
I have a similar issue here with Ubuntu.

### Description

Mails from a Microsoft Office 365 account doesn't show the attached pdf files in Nylas.
But the attached files show up in the web-client.

The sender used an installed Microsoft Outlook 2013 and sent the file
by "right-click - send to email" in the context menu of Windows 7.
The email had an empty message body and the pdfs file name as the subject line.

I use Nylas N1, version 0.4.40 installed on Ubuntu 16.04 LTS from the apt.nylas.com/ubuntu PPA.

I tried the following, but **couldn't reproduce**:

- Sending with and without message body
- Sending from Nylas and from web clients.
- Sending from Microsoft Office 365, MS Exchange and Google-Mail.
- Sending long and short file names.

Unfortunately, I don't own Windows 7 and Outlook 2013, so I couldn't test the *hot case*.

### Expected behavior

Nylas N1 shows file attachment with paperclip symbol in mail overview and the attached pdf file to download while viewing email.

### Actual behavior

Nylas N1 doesn't show the attached pdf file with a paperclip symbol or the actual file while viewing mail. But attachment is present, downloadable and viewable with a pdf reader when accessing the mail account by a browser.

### Version

- Nylas N1 0.4.40 from apt.nylas.com PPA
- Operating system: Ubuntu Linux 16.04 LTS
- Kernel: 4.4.0-22-generic #40-Ubuntu SMP
- Architecture: x86_64 GNU/Linux

**Comment by pscheid92** [comment]
I have done further testing and finally are able to **reproduce** the issue with the following steps:

1. Sending a email from a Microsoft Online account to a Microsoft Online account.
2. Add attachment with at least one "german umlaut" (äöü) in file name.

I tested some cases with a PDF file named "Bezüge.pdf" from GMail and Exchange to Microsoft Online and from Microsoft Online to GMail and Exchange without any issues. 

I also tested sending "Bezüge.pdf" from Nylas and from browser without any difference.
Empty or filled email body seems to make no difference, too.


@mfreminville I think Nylas version, operating system, used mail providers for sending and receiving would be interesting. Maybe there are some non-english-standard letters in your filenames, too?
